provisioning/gencdf.dhall

```diff
@@ -0,0 +1,52 @@
+let Prelude = ../cfgen/dhall/Prelude.dhall
+let T = ../cfgen/dhall/types.dhall
+
+let NodeInfo =
+      { hostname : Text
+      , data_iface : Text
+      , data_iface_type : T.Protocol
+      , io_disks : List Text
+      }
+
+let toNodeDesc
+    : NodeInfo -> T.NodeDesc
+    =     \(n : NodeInfo)
+      ->  { hostname = n.hostname
+          , data_iface = n.data_iface
+          , data_iface_type = Some n.data_iface_type
+          , m0_clients = { other = 3, s3 = 11 }
```

Comment:
@mssawant It's a stub now. 

BTW do you have an idea how many of them should be placed here? Do we need to request a new key to appear in ConfStore for `other`s?

Reply:
Presently we reserve 2 m0_clients of other types. This will be some number until we support dynamic addition of clients. So even 3 is fine because there's no good reason to select a number.